README.md

```diff
@@ -183,3 +183,17 @@ __Runtime network setup with CNI__
 4. Create bridge, TAP, and link all together with network interface previously created ([code](https://github.com/containers/virtcontainers/blob/0.5.0/network.go#L123-L205))
 
 5. Start VM inside the netns and start the container ([code](https://github.com/containers/virtcontainers/blob/0.5.0/api.go#L66-L70))
+
+## Storage
```

Comment:
Thanks for updating the doc here. Could you also add the limitation that devicemapper block device can only be used when creating a new pod. Adding a new container (with a devicemapper rootfs) after we started the VM won't work.